ApplicationLibCode/Application/Tools/RiaFieldHandleTools.cpp

```diff
@@ -48,3 +48,19 @@ void RiaFieldhandleTools::disableWriteAndSetFieldHidden( caf::PdmFieldHandle* fi
         fieldHandle->xmlCapability()->setIOWritable( false );
     }
 }
+
+//--------------------------------------------------------------------------------------------------
+///
+//--------------------------------------------------------------------------------------------------
+void RiaFieldhandleTools::updateOverrideStateAndLabel( caf::PdmFieldHandle* fieldHandle,
```

Comment:
Filenamer is RiaFieldHandleTools.cpp, while class name is RiaFieldhandleTools with lowercase h. Class name should probably have uppercase H, too?